lib/files.cf

```diff
@@ -1936,6 +1936,31 @@ body delete tidy
       rmdirs   => "true";
 }
 
+##-------------------------------------------------------
+## fsattrs
+##-------------------------------------------------------
+
+body fsattrs immutable
+# @brief Ensure promised file is immutable
```

Comment:
Humm, maybe this should be immutable(true_or_false)

so you could fsatters => immutable("true");

could then have additional bodies if desired for is_immutable and is_not_immutable.